include/template.php

```diff
@@ -272,16 +274,30 @@ function renderTemplate($view) {
 	// Check if we are using a PHP powered template or the standard one
 	$path = !empty($rep) ? $rep->getTemplatePath() : $config->getTemplatePath();
 	$path = $path . 'template.php';
-	if (is_readable($path)) {
+
+	if (is_readable($path)) 
+	{
 		$vars['templateentrypoint'] = $path;
 		executePlainPhpTemplate($vars);
-	} else {
+	}
+	else 
+	{
 		parseTemplate('header.tmpl');
 		flush(); // http://developer.yahoo.com/performance/rules.html#flush
 		parseTemplate($view . '.tmpl');
-		if ($view === 'directory' || $view === 'log') {
+		if ($view === 'directory' || $view === 'log') 
```

Comment:
I suggest an additional newline to distinguish from the former lines. More brackets as well:

> if (($view === 'directory') || ($view === 'log'))

Reply:
Done.